paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/MandateTest.kt

```diff
@@ -14,7 +14,7 @@ import org.robolectric.RobolectricTestRunner
 import org.robolectric.annotation.Config
 
 @RunWith(RobolectricTestRunner::class)
-@Config(sdk = [Build.VERSION_CODES.Q])
+@Config(sdk = [Build.VERSION_CODES.S])
```

Comment:
OOC, why did you have to change these?

Reply:
Tests were timing out... We don't know the root cause. This causes the test to use a different configuration, and causes them to pass. Basically anytime you see a `@Config(sdk = `, it's for this exact reason....